resource-detection-gcp/README.md

```diff
@@ -0,0 +1,65 @@
+# Resource detection Example
```

Comment:
not sure what this PR is about.

If it's about using the GCP resource detector - this is documented in https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-resource-providers-that-are-disabled-by-default

Reply:
This is just a small example that showcases what resource attributes are detected by the GCP resource detector.

This example also demonstrates the autoconfigure-spi in action, indicating that it can be used without the agent as well. 

The [current documentation](https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-resource-providers-that-are-disabled-by-default) contains instructions for automatic configuration.

Reply:
ok, got it.

I'd suggest you link to https://opentelemetry.io/docs/languages/java/automatic/configuration/#enable-resource-providers-that-are-disabled-by-default in the beginning of the doc - for users that want to use this detector.

Reply:
Just FYI: Our resource detector's [README page](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/gcp-resources#usage-with-auto-instrumentation) already links to these instructions.

But I will update this README as well.
Thanks!